.github/workflows/build_and_run_test_app_usb.yml

```diff
@@ -0,0 +1,72 @@
+name: Build and Run USB Test Application
+
+on:
+  schedule:
+    - cron: '0 0 * * SAT' # Saturday midnight
+  pull_request:
+    types: [opened, reopened, synchronize]
+    paths:
+      - 'usb/**'
+      - 'workflows/build_and_run_test_app_usb.yml'
```

Comment:
This section is not required anymore. In idf-extra-components we only wanted to run this pipeline only when changes it `usb` directory were made.

(The path on line 10 is incorrect, so the GH action is not run at all)